tests/utils/profilehandler/profile_handler.go

```diff
@@ -332,11 +344,97 @@ func GenerateClusterCreateFlags(profile *Profile, client *rosacli.Client) ([]str
 
 		clusterConfiguration.Autoscaler = autoscaler
 	}
+	if profile.ClusterConfig.NetWorkingSet {
```

Comment:
I think this could be typo error` NetWorkingSet`  -> `NetworkingSet`

Reply:
Thanks, updated